File: app/src/main/java/com/ryanrvldo/moviecatalogue/data/source/model/Episode.java
package com.ryanrvldo.moviecatalogue.data.source.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import lombok.Data;

@Data
public class Episode {
    @Expose
    private int id;

    @SerializedName("air_date")
    @Expose
    private String airDate;

    @SerializedName("episode_number")
    @Expose
    private int number;

    @Expose
    private String name;

    @Expose
    private String overview;

    @SerializedName("still_path")
    @Expose
    private String stillPath;
}

File: app/src/main/kotlin/com/ryanrvldo/moviecatalogue/data/model/Episode.kt
package com.ryanrvldo.moviecatalogue.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Episode(
    @Expose val id: Int,
    @SerializedName("air_date") @Expose val airDate: String,
    @SerializedName("episode_number") @Expose val number: Int,
    @Expose val name: String,
    @Expose val overview: String,
    @SerializedName("still_path") @Expose val stillPath: String
)